src/frontend/src/icp/utils/wallet.utils.ts

```diff
@@ -0,0 +1,7 @@
+import { initIcpWalletWorker } from '$icp/services/worker.icp-wallet.services';
+import { initIcrcWalletWorker } from '$icp/services/worker.icrc-wallet.services';
+import type { IcToken } from '$icp/types/ic';
+import type { InitWallerWorkerParams, WalletWorker } from '$lib/types/listener';
+
+export const initWalletWorker = async ({ token }: InitWallerWorkerParams): Promise<WalletWorker> =>
```

Comment:
nitpick: function does not need to be marked as `async`

Reply:
Indeed!